pennylane/_device.py

```diff
@@ -154,6 +156,12 @@ def wire_map(self):
         the wire labels used on this device"""
         return self._wire_map
 
+    @property
+    def execution(self):
```

Comment:
Just wondering if `execution` is the best choice of word, with an alternative being `num_executions` (we already have `num_wires`), or something shorter.

Reply:
I'm a fan of `num_executions`, `execution_count`, or even `executions` (making it plural seems to make it clearer)

Reply:
A logical suggestion! Changes have been made.